### PR TITLE
[plugin.video.putio] 2.3.0

### PR DESCRIPTION
**Pull request by cenkalti**
### Description
* Add support for loading all available subtitles
### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.putio/addon.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <extension point="xbmc.addon.metadata">
         <summary lang="en">Put.io for Kodi</summary>
         <description lang="en">Put.io is a service that combines cloud storage and bittorrent. You need an account to use this plug-in. Get one from https://put.io</description>
-        <disclaimer></disclaimer>
+        <disclaimer lang="en">Please read terms of service at https://put.io/terms-of-service/</disclaimer>
```
Comment (enen92) [inline plugin.video.putio/addon.xml]: en_GB

**Comment by cenkalti** [comment]
@enen92 The exception in build log is unrelated. Is there a way to retrigger Travis build?

**Comment by cenkalti** [comment]
Hello @enen92. I can't see the latest version when I want to add putio add-on from the repository. How long does it take the repository to get updated after merge?

**Comment by Rechi** [comment]
@cenkalti this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Comment by enen92** [comment]
@cenkalti the repository sync script runs twice a day iirc. But rechi knows better.

**Comment by Rechi** [comment]
Repository is synced after 2 hours and 30 minutes, but Kodi only checks the repo every 24 hours.

**Pull request merged**